### DISCUSSION TITLE
Signal light for Cargo Scooter

### DISCUSSION THREAD

**Discussion by Prolbo** [FakeFishGames/Barotrauma#10936] [Ideas and suggestions]
Would be nice to have signal light (it actually can be blinking pixel with no light at all) on the cargo scooter, so player can find it after dismounting and leaving to fight / mine / explore. Bonus points if said light works only after cargo scooter is left underwater for some time, say 10 seconds.

**Comment by Stallfighter** [comment]
Not a bad idea actually, instead of wasting glowstick or alerting wildlife with sonar beacon inside it.

**Comment by PiatraAlatyr** [comment]
It's a good idea to build an analogue of emergency lighting into a scooter. It is quite difficult to find it in dark water.
